stable/metrics-server/templates/metric-server-service.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "metrics-server.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- with .Values.service.labels }}
```

Comment:
Can we format this properly here?

Reply:
I've updated them all, I think this is what we want here.